### DISCUSSION TITLE
Server side security improvements

### DISCUSSION THREAD

**Discussion by MaybeThisIsRu** [vercel/vercel#4707] [Ideas] [answered]
Hi all, 

I ran a test on [Mozilla's Observatory](https://observatory.mozilla.org/) (bless Mozilla) and received the following results for a static site:

![Screenshot from 2020-06-24 11-59-24](https://user-images.githubusercontent.com/3974019/85508775-55995480-b612-11ea-87e0-56d971a4d2cf.png)

Are any of these items feasible to be tackled at the scale/style Vercel operates on/in?

**Comment by styfle** [comment, marked as answer]
Yes, you can create a `vercel.json` in the root of your project to add headers to any path including static files.

See the [`headers`](https://vercel.com/docs/configuration#project/headers) property for more info.

Also see this other article about [enabling CORS](https://vercel.com/knowledge/how-to-enable-cors) for functions.

**Reply by MaybeThisIsRu** [reply]
Whoops, not sure how I missed that. Thank you!

**Comment by iinfin** [comment]
@hirusi I've been using [this configuration](https://github.com/u29dc/hbp/blob/master/example/vercel.json) to get [130/100](https://observatory.mozilla.org/analyze/iinf.in) if that helps.
(There is another 5 points possible but I don't remember how to get it, should be something [in here](https://github.com/mozilla/http-observatory/blob/master/httpobs/scanner/grader/grade.py))

**Reply by MaybeThisIsRu** [reply]
Hey, thank you. I've already managed to hit a 120ish last I remember and am happy with where it is.
